samples/workload-simulator.html

```diff
@@ -794,7 +789,6 @@ <h2><center>Web graphics workload simulator</center></h2>
             usage: GPUBufferUsage.COPY_SRC | GPUBufferUsage.MAP_WRITE,
             mappedAtCreation: true,
           });
-          buffer.size = mapAsyncArray.byteLength;
```

Comment:
I did something different here because it seemed like `.size` wasn't the actual size of the buffer, but the size of the section of it that was being used. I'm not sure if this was actually different or not?
https://github.com/gpuweb/gpuweb/pull/4136/files